### PR TITLE
8GadgetPack.net.8GadgetPack version 36.0.0

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/100052)

### CODE REVIEW

File: manifests/8/8GadgetPack/net/8GadgetPack/36.0.0/8GadgetPack.net.8GadgetPack.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: 8GadgetPack.net.8GadgetPack
```
Comment (Trenly) [inline manifests/8/8GadgetPack/net/8GadgetPack/36.0.0/8GadgetPack.net.8GadgetPack.installer.yaml:4]:
```suggestion
PackageIdentifier: 8GadgetPack.8GadgetPack
```
Package folder will need to be updated when identifier is changed

File: manifests/8/8GadgetPack/net/8GadgetPack/36.0.0/8GadgetPack.net.8GadgetPack.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: 8GadgetPack.net.8GadgetPack
```
Comment (Trenly) [inline manifests/8/8GadgetPack/net/8GadgetPack/36.0.0/8GadgetPack.net.8GadgetPack.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: 8GadgetPack.8GadgetPack
```
Package folder will need to be updated when identifier is changed

File: manifests/8/8GadgetPack/net/8GadgetPack/36.0.0/8GadgetPack.net.8GadgetPack.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: 8GadgetPack.net.8GadgetPack
```
Comment (Trenly) [inline manifests/8/8GadgetPack/net/8GadgetPack/36.0.0/8GadgetPack.net.8GadgetPack.yaml:4]:
```suggestion
PackageIdentifier: 8GadgetPack.8GadgetPack
```
Package folder will need to be updated when identifier is changed

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Manifest validation error:
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\8\8GadgetPack\net\8GadgetPack\36.0.0', expected 'manifests\8\8GadgetPack\8GadgetPack\36.0.0'

There's a `\net` in the file path that needs to be removed.

**Comment by Trenly** [comment]
Close with reason: Stale PR;

**Pull request closed without merging**